templates/bmc.yml.erb

```diff
@@ -1,15 +1,21 @@
 ---
 # BMC management (Bare metal power and bios controls)
 :enabled: <%= @module_enabled %>
+
 # Available providers:
 # - freeipmi / ipmitool - requires the appropriate package installed, and the rubyipmi gem
 # - shell - for local reboot control (requires sudo access to /sbin/shutdown for the proxy user)
+# - ssh - limited remote control (status, reboot, turn off)
 :bmc_default_provider: <%= scope.lookupvar("foreman_proxy::bmc_default_provider") %>
 <% if scope.lookupvar("foreman_proxy::bmc_default_provider") == 'ssh' -%>
+
+# default user and ssh key for ssh provider
 :bmc_ssh_user: <%= scope.lookupvar("foreman_proxy::bmc_ssh_user") %>
 :bmc_ssh_key: <%= scope.lookupvar("foreman_proxy::bmc_ssh_key") %>
-:bmc_ssh_powerstatus: "<%= scope.lookupvar("foreman_proxy::bmc_ssh_powerstatus") %>"
-:bmc_ssh_powercycle: "<%= scope.lookupvar("foreman_proxy::bmc_ssh_powercycle") %>"
-:bmc_ssh_poweroff: "<%= scope.lookupvar("foreman_proxy::bmc_ssh_poweroff") %>"
-:bmc_ssh_poweron: "<%= scope.lookupvar("foreman_proxy::bmc_ssh_poweron") %>"
+
+# individual commands for actions
+:powerstatus: "<%= scope.lookupvar("foreman_proxy::bmc_ssh_powerstatus") %>"
+:powercycle: "<%= scope.lookupvar("foreman_proxy::bmc_ssh_powercycle") %>"
+:poweroff: "<%= scope.lookupvar("foreman_proxy::bmc_ssh_poweroff") %>"
+:poweron: "<%= scope.lookupvar("foreman_proxy::bmc_ssh_poweron") %>"
```

Comment:
I'd rather suggest to keep the `bmc_ssh` prefix and change the core. The added tests are still good addition IMHO.